### ISSUE TITLE
Does the bot incubate eggs?

### ISSUE THREAD

**Issue by FlyingFish10** [finndev/PokeBuddy#344]

**Comment by RobJansen** [comment]
Yes it does

**Comment by FlyingFish10** [comment]
Will it do it automatically?

**Comment by Juan44** [comment]
How!!??? How i incube eggs in the bot

**Comment by IamDeric** [comment]
no

**Comment by RobJansen** [comment]
It works if you walk 15 kmph

**Comment by barbaricacidman** [comment]
so from this post im assuming you have to log into pokemon on phone and incubate your eggs. is there a safe way of doing it? my concern is im in UK and bot is set in manhatten!

**Comment by curtiola** [comment]
If you know how to spoof your location you can match the bot to sign in and start incubation, but the devs have noted that they will add this eventually, it got kinda buried these last few days, see #172

**Comment by barbaricacidman** [comment]
yeah i understand. im on iphone so dont think i can spoof without jailbreaking. ill just wait. 

thanks

**Comment by barbaricacidman** [comment]
any thoughts on turning off location services while logging into the app on phone? in theory it wont know where you are

**Comment by curtiola** [comment]
Yeah that works perfectly
